### DISCUSSION TITLE
Undefined Method

### DISCUSSION THREAD

**Discussion by dev-techguy** [livewire/livewire#3788] [7. Bugs (archived)] [answered]
Call to undefined method Symfony\Component\HttpFoundation\BinaryFileResponse::withHeaders() {"exception":"[object] (Error(code: 0): Call to undefined method Symfony\Component\HttpFoundation\BinaryFileResponse::withHeaders() at /var/www/pam.easyncpay.com/vendor/livewire/livewire/src/DisableBrowserCache.php:21)

**Comment by joshhanley** [comment, marked as answer]
@dev-techguy there was some issues with file downloads which has been fixed and tagged a new release 2.6.4.

Can you please test the latest version and let us know if it has fixed your issue or not?

Thanks!

**Reply by dev-techguy** [reply]
Yeah working. Thanks
